### ISSUE TITLE
Display Map To User

### ISSUE THREAD

**Issue by Reltre** [Public-Tree-Map/public-tree-map#13]
This is work that needs to be done for the Map component of this application.  The map component should default to the city of Santa Monica and meet the following requirements:

1. Display all trees within view.
2. Allow for pan and zoom.

Filters and display of heritage trees are a separate task. The aim of this card is to create a Map component that has the core functionality needed for street tree map: a component that can be expanded upon if needed.

Note: This card is blocking, and must be completed before cards related to individual tree selection can be completed.

Resources:

https://github.com/PaulLeCam/react-leaflet
https://react-leaflet.js.org/docs/en/intro.html
